test/e2e/ingress.go

```diff
@@ -119,6 +125,8 @@ var _ = framework.KubeDescribe("Loadbalancing: L7 [Feature:Ingress]", func() {
 		})
 
 		It("shoud create ingress with given static-ip ", func() {
+			cloudConfig := framework.TestContext.CloudConfig
```

Comment:
this should be encapsulated in the gceController struct https://github.com/kubernetes/kubernetes/blob/master/test/e2e/ingress_utils.go#L910